core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -150,23 +150,25 @@ THE SOFTWARE.
     <script src="${resURL}/jsbundles/sortable-drag-drop.js" type="text/javascript"/>
     <script src="${resURL}/jsbundles/app.js" type="text/javascript" defer="true" />
   </head>
-  <body id="jenkins" class="${layoutType} jenkins-${h.version}" data-version="${h.version}" data-model-type="${it.class.name}">
+  <body id="jenkins" class="${layoutType} jenkins-${h.version}" data-version="${h.version}" data-model-type="${it.class.name}"
+        data-search-url="${rootURL + '/search/suggest'}"
+        data-search-help-url="${%searchBox.url}">
     <l:command-palette />
 
     <j:if test="${layoutType!='full-screen'}">
       <!-- for accessibility, skip the entire navigation bar and etc and go straight to the head of the content -->
       <a href="#skip2content" class="jenkins-skip-link">Skip to content</a>
 
+      <j:set var="breadcrumbs">
+        <j:set var="mode" value="breadcrumbs" />
+        <d:invokeBody />
+      </j:set>
+
       <l:pageHeader title="${%title}"
                     logoAlt="${%jenkinshead.alt}"
                     searchPlaceholder="${%search}"
                     searchHelpUrl="${%searchBox.url}"
                     logout="${%logout}"/>
```

Comment:
This looks like it renders `l:pageHeader` with a non-`header` `mode` (previously, see line 139), which would be a breaking change.

Reply:
I created a branch in a small sample plugin to show the problem https://github.com/scherler/layout-header-plugin/tree/header_redesign

Reply:
Hey, I've updated it to _hopefully_ fix this. I've moved the loading of breadcrumbs above `<j:set var="mode" value="header" />`, the new header works as expected and the plugin now loads the header JS in.